packages/suite/src/components/wallet/PrivacyAccount/SliderInput.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { forwardRef, useEffect, useImperativeHandle, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { Input, InputProps, variables } from '@trezor/components';
+
+const LevelContainer = styled.div`
+    position: absolute;
+    right: 20px;
+    width: 68px;
+
+    .inputAddon {
+        font-size: ${variables.FONT_SIZE.SMALL};
+        font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+        color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+        pointer-events: none;
+    }
```

Comment:
I would prefer to style the component that is passed to FeeInput in innerAddon props. We should avoid styling child components by class names - it's too tight coupling. (I would use it only if there is no other way - like if you need to style some not very well written 3rd party component...)

Reply:
The most important part is ` pointer-events: none;`, which would not work if I style the inner `<span/>` with it :/ Perhaps a better way is to use [the Modal approach. ](https://github.com/trezor/trezor-suite/blob/f8bac66a94ce1eb062fab5c2f1379abf1b7dc773/packages/components/src/components/Modal/index.tsx#L320) While I agree that class names are too coupling, I believe in component libraries they still [can be used](https://mui.com/material-ui/customization/how-to-customize/#overriding-styles-with-class-names) [for styling](https://react-select.com/styles#inner-components) inner elements.

Reply:
Oh, I see, in that case we can do the dependency explicit, by exporting `InputAddon` component from Input, import it here and  use
```suggestion
    ${InputAddon} {
        font-size: ${variables.FONT_SIZE.SMALL};
        font-weight: ${variables.FONT_WEIGHT.MEDIUM};
        color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
        pointer-events: none;
    }
```

Reply:
625eef590